sensor_msgs_py/sensor_msgs_py/point_cloud2.py

```diff
@@ -144,40 +145,6 @@ def read_points_list(cloud, field_names=None, skip_nans=False, uvs=[]):
                                                 skip_nans, uvs)]
 
 
-def create_cloud(header, fields, points):
```

Comment:
Isn't this part of the public API? This shouldn't be removed, unless I'm missing something.

Reply:
You are right. I just added the code I used locally. I will adapt it to satisfy all of the existing interfaces.

Reply:
I added it again.